RELEASE/scripts/autoscend/paths/dark_gyffte.ash

```diff
@@ -363,16 +363,31 @@ boolean bat_shouldPickSkills(int hpLeft)
 	return false;
 }
 
+boolean bat_haveEnsorcelee() // checks if you have a current Ensorceled Monster
+{
+	if(!auto_have_skill($skill[Ensorcel]))
+		return false;
```

Comment:
```suggestion
	if(!auto_have_skill($skill[Ensorcel]))
	{
		return false;	//in case mafia does not clear ensorcelee property when you change your skills and drop Ensorcel.
	}
```
add comment and match the formatting we use elsewhere